privacy/scrubber.go

```diff
@@ -81,6 +83,103 @@ func NewScrubber() Scrubber {
 	return scrubber{}
 }
 
+func (scrubber) ScrubRequest(bidRequest *openrtb2.BidRequest, enforcement Enforcement) *openrtb2.BidRequest {
+	var userExtParsed map[string]json.RawMessage
+	userExtModified := false
+
+	var userCopy *openrtb2.User
+	if bidRequest.User != nil {
+		userCopy = ptrutil.Clone(bidRequest.User)
+	}
```

Comment:
Nitpick: The copies can simply be `userCopy := ptrutil.Clone(bidRequest.User)`. No special handling of nil is required.